tests/qos/test_tunnel_qos_remap.py

```diff
@@ -68,8 +71,10 @@ def _last_port_in_last_lag(lags):
     return lags[last_lag][-1]
 
 
-def test_encap_dscp_rewrite(ptfhost, upper_tor_host, lower_tor_host,                            # noqa F811
-                            toggle_all_simulator_ports_to_lower_tor, tbinfo, ptfadapter, tunnel_qos_maps):       # noqa F811
+@pytest.mark.enable_active_active
+def test_encap_dscp_rewrite(ptfhost, upper_tor_host, lower_tor_host,  # noqa F811
```

Comment:
maybe it is better to add a new fixture to setup the active-active ports into active-standy mode.

Reply:
Added common fixtures for the different use cases present in this file.